reference/tax_settings.v3.yml

```
@@ -178,6 +190,18 @@ components:
           default: true
           type: boolean
           description: This setting applies only if a merchant enters tax-inclusive prices. When enabled, the store subtracts the itemʼs store tax rate before calculating tax using the shopperʼs tax zone. The tax-exclusive amount will be the same across all tax zones. When disabled, the tax-inclusive price remains the same across all tax zones; only the tax amount will vary based on the shopperʼs location. The tax-exclusive amount may vary among tax zones. These calculations are relevant for tax pricing and tax quotations that use basic tax.
+        should_use_geolocation_to_determine_guest_shopper_tax_zone:
+          default: false
+          type: boolean
+          description: When enabled, geolocation data will be used to match the guest shopper with a relevant country-level tax zone for tax estimation on the storefront. The geolocation data will be used to determine the corresponding tax zone provided in basic tax - once the corresponding tax zone is found, the tax estimation will be applied to the store. When this setting is disabled, the tax estimation will be based on the guest shopper tax zone, the guest shopper tax zone can be reconfigured by updating guest_shopper_tax_zone_id or store_tax_zone_id.
+        guest_shopper_tax_zone_id:
+          default: 1
+          type: integer
+          description: The setting allows the user to configure the guest shopper tax zone for shoppers to see in the storefront. The user can configure the tax zone id to any existing tax zone in basic tax where the chosen tax zone will be used for tax estimation. When used in conjunction with store_tax_zone_id and should_subtract_store_tax, the rules of should_subtract_store_tax_will apply for the storefront where the tax rates will be subtracted and recalculated based on the two tax zones provided.
```

```suggestion
          description: Describes which tax zone is used for guest shoppers when requiring storefront tax estimation. When applicable, the shopper's geolocation data will be used in favour of the guest shopper tax zone.
```

It's an interesting challenge to try and provide "just enough" detail, especially since there is a matrix of relationships now. I'm hoping we can limit the store tax subtraction topic to those settings.

---

👍